integreat_cms/cms/views/pages/page_form_view.py

```diff
@@ -416,6 +417,9 @@ def post(self, request, *args, **kwargs):
                 # Add the success message
                 page_translation_form.add_success_message(request)
 
+            invalidate_obj(page_form.instance)
+            invalidate_obj(page_translation_form.instance)
```

Comment:
This is to avoid Errors by cache.

Reply:
What errors exactly? The form instances should already be invalidated when the forms are saved. If that doesn't work, we have a bigger problem.

Reply:
I saw something like `Key = (...) already exists` but cannot reproduce it 🤔 I'll remove those lines.